packages/relay/src/utils.ts

```diff
@@ -147,4 +149,35 @@ export class Utils {
       accountId: AccountId.fromString(operatorId.trim()),
     };
   }
+
+  /**
+   * Get the consensus node version
+   */
+  public static async getConsensusNodeVersion(): Promise<string> {
```

Comment:
I'd move this to `hedera.ts` given is specific to that namesapce

Reply:
Moved.